libraries/node-core-library/src/JsonFile.ts

```diff
@@ -113,10 +113,14 @@ export class JsonFile {
   /**
    * Loads a JSON file.
    */
-  public static load(jsonFilename: string): JsonObject {
+  public static load(jsonFilename: string, validateStrictJsonFormat?: boolean): JsonObject {
```

Comment:
Only the `load` function was modified to show the package.json parsing validation change. If async and callback variants need to be changed as well, please comment.